apps/music/test/unit/metadata/id3v1_test.js

```diff
@@ -26,21 +27,34 @@ suite('id3v1 tags', function() {
     navigator.getDeviceStorage = RealGetDeviceStorage;
   });
 
-  suite('simple id3v1', function() {
-
-    test('id3v1', function(done) {
-      parseMetadata('/test-data/id3v1-simple.mp3').then(function(metadata) {
-        done(function() {
-          assert.strictEqual(metadata.tag_format, 'id3v1');
-          assert.strictEqual(metadata.artist, 'AC/DC');
-          assert.strictEqual(metadata.album, 'Dirty Deeds Done Dirt Cheap');
-          assert.strictEqual(metadata.title, 'Problem Child');
-          assert.strictEqual(metadata.tracknum, 5);
-          assert.strictEqual(metadata.trackcount, undefined);
-        });
+  test('id3v1', function(done) {
+    parseMetadata('/test-data/id3v1-simple.mp3').then(function(metadata) {
+      done(function() {
+        assert.strictEqual(metadata.tag_format, 'id3v1');
+        assert.strictEqual(metadata.artist, 'AC/DC');
+        assert.strictEqual(metadata.album, 'Dirty Deeds Done Dirt Cheap');
+        assert.strictEqual(metadata.title, 'Problem Child');
+        assert.strictEqual(metadata.tracknum, 5);
+        assert.strictEqual(metadata.trackcount, undefined);
       });
     });
+  });
 
+  test('no metadata', function(done) {
+    fetchBuffer('/test-data/no-tag.mp3').then(function(buffer) {
+      var blob = new Blob([buffer]);
+      blob.name = 'no-tag.mp3';
+      return blob;
+    }).then(parseMetadataBlob).then(function(metadata) {
+      done(function() {
```

Comment:
Why do you need to use fetchBuffer and parseMetadataBlob in this test instead of just using parseMetadata like you do in the id3v1 test above?